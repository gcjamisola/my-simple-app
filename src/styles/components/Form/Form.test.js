import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import {Form} from './index';

test('it works', () => {
	const tree = renderer.create(<Form/>).toJSON();
	expect(tree).toMatchSnapshot();
});