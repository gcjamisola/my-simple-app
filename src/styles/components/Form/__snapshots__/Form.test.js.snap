// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  max-width: 400px;
  margin: 50px auto;
  background: #fff;
  border-radius: 2px;
  padding: 20px;
}

.c0 ul {
  list-style: none;
  padding: 0;
  margin: 0;
}

.c0 li {
  display: block;
  padding: 9px;
  border: 1px solid #DDDDDD;
  margin-bottom: 30px;
  border-radius: 3px;
}

.c0 li:last-child {
  border: none;
  margin-bottom: 0px;
  -webkit-text-align: center;
  text-align: center;
}

.c0 li > label {
  display: block;
  float: left;
  margin-top: -19px;
  background: #FFFFFF;
  height: 14px;
  padding: 2px 5px 2px 5px;
  color: #9e9e9e;
  font-size: 14px;
  overflow: hidden;
}

.c0 li > span {
  background: #ffe5bd;
  display: block;
  padding: 3px;
  margin: 0 -9px -9px -9px;
  -webkit-text-align: center;
  text-align: center;
  color: #ff9400;
  font-size: 11px;
}

.c0 input[type="text"],
.c0 input[type="date"],
.c0 input[type="datetime"],
.c0 input[type="email"],
.c0 input[type="number"],
.c0 input[type="search"],
.c0 input[type="time"],
.c0 input[type="url"],
.c0 input[type="password"],
.c0 textarea,
.c0 select {
  box-sizing: border-box;
  width: 100%;
  display: block;
  outline: none;
  border: none;
  height: 25px;
  line-height: 25px;
  font-size: 16px;
  padding: 0;
}

.c0 input[type="text"][name="street"],
.c0 input[type="text"][name="city"],
.c0 input[type="text"][name="zipcode"] {
  border-bottom: 1px solid rgba(0,0,0,.25);
  margin-bottom: 10px;
}

.c0 input[type="text"]::-webkit-input-placeholder {
  font-size: 10px;
  color: rgba(0,0,0,.5);
}

.c0 input[type="text"]::-moz-placeholder {
  font-size: 10px;
  color: rgba(0,0,0,.5);
}

.c0 input[type="text"]:-ms-input-placeholder {
  font-size: 10px;
  color: rgba(0,0,0,.5);
}

.c0 input[type="text"]::placeholder {
  font-size: 10px;
  color: rgba(0,0,0,.5);
}

<form
  className="c0"
/>
`;
